docs/cave/d2d-gis-shapefiles.md

```diff
@@ -1,27 +1,35 @@
-* **Load As**: In this section the shapefiles can be loaded as a Map or as a Product. Descriptions of both follow.
-    * **Map**: When this radio button is selected, the selected shapefile displays as a map. Click and hold the middle mouse button to open a pop-up menu and select **Show Map Legend** to display the **Map ID** in the **Legend area** at the bottom of the display.
-    * **Product**: When this radio button is selected, the start and end date/time data fields become enabled. You can select a start and end date/time from the **Calendar** dialog.   Clicking on the Calendar icon opens the **Calendar** dialog. Loading a shapefile with the **Product** radio button selected will return a shaded (color-filled) image of the Map Product. The **Product ID** will appear in the Legend area at the bottom of the display after selecting **Show Product Legend** from the middle mouse click pop-up menu. The map product displays with the model data when the valid time of the model data falls within the start/end time range selected for the map product in the **GIS DataStore Parameters** dialog. When the valid time of the model data falls outside the start/end time of the map product, the map product image does not display.
+* **DataStore Type**: You can select a file type from the dropdown list. The only option is *GIS File*.
+* **Connection Parameters**:  Click the **Browse** button and navigate to the directory where your shapefiles are. Pressing **Connect** will populate the available shapefiles.
+* **Load As**: Shapefiles can be loaded as a Map or as a Product. 
+    * **Map**: The selected shapefile displays as a map, similar to if you load a map from the Maps menu. 
+    * **Product**: When this radio button is selected, you will also need to select the start and end date/time the data is valid for. The selected shapefile displays as a product with a shaded (color-filled) image. When plotting with additional products, if the display time falls within the start/end time range selected, the shapefile will display. When the valid time falls outside the start/end time, the map product image does not display.
+* **Table**:  This section lists all of the available shapefiles that are available for display.
 
-* **Table**:  In this section you can select one of the shapefiles listed in the window for display. The selected file will be highlighted and the OK button at the bottom of the dialog will be enabled. Clicking the OK button closes the GIS DataStore Parameters dialog box and displays the selected map or map product.
+![image](../images/gis-gui.png)
+
+---
 
 ## GIS Data Preferences
 
-Using the **Preferences** dialog, you can modify certain GIS display attributes in CAVE without having the GIS application open. Access the GIS Viewer via **CAVE** > **Preferences** > **GIS Viewer**.
+Updating GIS display preferences is accessed through **CAVE** > **Preferences** > **GIS Viewer**.
```

Comment:
The only thing I might add here (I know it shows it all in the image), but maybe one quick sentence about the Preferences you're able to alter are highlight color, highlight style, highlight width and product opacity, just in case someone wants to try and do a "find" on that text (they won't find it from the image)?